src/java.base/share/native/libjli/jli_util.h

```diff
@@ -81,14 +81,23 @@ JLI_GetAppArgIndex();
 #define JLI_StrCSpn(p1, p2)     strcspn((p1), (p2))
 #define JLI_StrPBrk(p1, p2)     strpbrk((p1), (p2))
 
+/*
+ * Visual Studio 2015 and above, alongside Windows 10 and the UCRT
+ * now use the C99 snprintf, which is no longer identical to the
+ * outdated Windows _snprintf, so we can remove the compatibility
+ * workaround.
+ * https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference
+ * /snprintf-snprintf-snprintf-l-snwprintf-snwprintf-l?view=msvc-170
+ */
```

Comment:
I don't think the comment about the *lack* of a workaround is needed, just adding clutter.  But this isn't code I have much involvement with.  Other than that, the change looks fine.

Reply:
I agree, we don't document the absence of a workaround.